R/reportPrices.R

```diff
@@ -333,7 +337,7 @@ reportPrices <- function(gdx,output=NULL,regionSubsetList=NULL,t=c(seq(2005,2060
   tmp = compute_agg_price_fe(tmp,output,"Transport")
   
   #Final energy prices
-  a = abs(mselect(balfinen.m[,y,][rbind(finenbal,fe2es)]))/abs((budget.m+1e-10)) # Translate the marginal utility of the constraint into the marginal income (price)
```

Comment:
This part only adds the FE taxes for the stationary sectors. ``prices_fe_bi`` only is used for stationary prices not for transport prices. Running edge_esm where fe2es has additional entries breaks the code here. Anyways, I think we will overhaul the reportPrices completely in a few weeks. Because as far as I understand the taxes should now be already in the fe marginals (these are the ``w/ CO2 cost`` price variables). We do not need these parts anymore then. However, the consistency of the FE prices will still need to be checked more thoroughly.

Reply:
And addition: fe2es might be required here if we run buildings services_putty. However, as we are currently still trying to get this running, it should not bother now and we can take care of this later.